### ISSUE TITLE
Selecting project from 'git' panel stops Code from tracking changes

### ISSUE THREAD

**Issue by ultraviolet-1986** [alefragnani/vscode-project-manager#399]
By selecting a project from the extension's 'git' pane (after detecting and adding projects automatically), VSCode is no longer displaying git changes by highlighting a file yellow in the files explorer pane and often the source control panel needs to be refreshed when changes are made. Timeline changes are also not shown and changes cannot be opened in another panel.

By opening a folder and saving it as a project manually in the 'Favourites' pane (using this extension still), the git functionality is restored and works correctly.

This affects Code-Insiders as well as current version of Code. Using Fedora Silverblue.

Thank you for your time, extension is a must ^_^b

**Comment by ultraviolet-1986** [comment]
On another machine, this behaviour cannot be reproduced. It appears to actually be a problem with VS Code itself affecting 'git' changes. ATM, I get:

`Warning: Failed to watch ref '/var/home/william/Documents/Workspace/git/<repo>/.git/refs/remotes/origin/test', is most likely packed.`

The behaviour in my first post is entirely Code affecting the extension and not the extension affecting Code. This *can* be reproduced and does not change on cloning the same repo again.

Looks like I wasted your time, sorry. Closing this.
